example-client/README.md

```diff
@@ -3,11 +3,219 @@ Example CoAP client
 
 This is an example coap client application, that can be use with any LwM2M Server.
 
+# Running
+
 Usage
 -----
 
     ./run.sh <registration-url> [<keystore-file>]
 
+note: parameters are parsed like a [www URL query string](https://en.wikipedia.org/wiki/Query_string) where ? and & are delimeters for the endpoint name and account ID
+
 *For example:*
-        
-    ./run.sh 'coap://localhost:5683/rd?ep=device007&lt=3600&b=U'    
+    
+    cd java-coap/example-client
+
+    ./run.sh "coaps://lwm2m.us-east-1.mbedcloud.com:5684/rd?ep=${ENDPOINT_NAME}&aid=${ACCOUNT_ID}" ../../keystore/example-client-device-keystore.jks
+
+<img src="images/example-running.png" width="600">
+
+    *note: replace ${ENDPOINT_NAME} and ${ACCOUNT_ID} with the value defined in the Configuring section below*
+
```

Comment:
- Styling: Again, bold the word note and start it with capital letter. 
- Styling: add ` characters around the parameters (or whatever they are ${...)
- Linking: in the [Configuration](add here first hash and then the section title, no spaces) section. (leave out the word below)